python/README.md

```diff
@@ -0,0 +1,187 @@
+## Getting started
+This module contains the building blocks to build new Flow connectors in Python, as well as adapting external Python connectors written in standard protocols to Flow. A Python connector is a subclass of the `Connector` classes in the `capture`, `derive`, and `materialize` (once it's written) submodules. Convenience classes `shim_airbyte_cdk.CaptureShim` and `shim_singer_sdk.CaptureShim` are provided to adapt existing Airbyte or Singer/Meltano connectors.
+
+A Python connector is composed of a few critical pieces:
+* `pyproject.toml` defines the Python module and its dependencies. For example:
+  ```toml
+  [tool.poetry.dependencies]
+  # If we're pulling in an open-source connector that doesn't need any changes, we can define it here. Subdirectory can be omitted if the connector lives at the root of the repo.
+  source_bigdata = { git = "https://github.com/big/data.git", subdirectory = "connector" }
+  # If we're working with an open-source connector that we have pulled in-tree, we can define it like this, where the connector was pulled into the subdirectory "upstream-connector"
+  source_bigdata = { path = "upstream-connector" }
+  ```
+* In addition to being useful documentation for other people running your connector, `test.flow.yaml` or some other valid Flow spec is needed to invoke your connector locally. You'll point `flowctl` at this file to test your connector when developing locally.
```

Comment:
The vision is that various `test.flow.yaml` files will become fully-automated acceptance tests for each connector, which is required for every connector and driven by our CI.

We would have actual working credentials (sops protected) which grab an actual set of data from the live API using the connector, that's then verified against an expectation.

More plumbing to do here, of course.